CommonMark/Parser/BlockMethods.cs

```diff
@@ -92,16 +93,355 @@ private static void BreakOutOfLists(ref Block blockRef, LineInfo line)
             {
                 while (container != null && container != b)
                 {
-                    Finalize(container, line);
+                    Finalize(container, line, settings);
                     container = container.Parent;
                 }
 
-                Finalize(b, line);
+                Finalize(b, line, settings);
                 blockRef = b.Parent;
             }
         }
 
-        public static void Finalize(Block b, LineInfo line)
+        static List<string> ParseTableLine(string line, StringBuilder sb)
+        {
+            var ret = new List<string>();
+
+            var i = 0;
+
+            if (i < line.Length && line[i] == '|') i++;
+
+            while (i < line.Length && char.IsWhiteSpace(line[i])) i++;
+
+            for (; i < line.Length; i++)
+            {
+                var c = line[i];
+                if (c == '\\')
+                {
+                    i++;
```

Comment:
Doesn't this just ignore all backslashes? I think that it should only ignore it when the next character is `|` and in this case the pipe character would be added to the result as a literal and not be parsed as cell separator.